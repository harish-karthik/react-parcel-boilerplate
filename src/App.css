body {
  font-family: "Barlow";
}
h1{
	font-weight: 500;
}
