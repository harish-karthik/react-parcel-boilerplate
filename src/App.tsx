import React from "react";
import Image from "jsx:./assets/logo.svg";
import svgUrl from "./assets/logo.svg";
import "./App.css";

export const App = () => {
  return (
    <>
      <h1>Hello World</h1>
      <img src={svgUrl} width={200} />
    </>
  );
};
