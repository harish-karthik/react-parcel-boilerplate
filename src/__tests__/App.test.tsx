import React from "react";
import renderer from 'react-test-renderer';
import {App} from "../App";

test('app component', () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
