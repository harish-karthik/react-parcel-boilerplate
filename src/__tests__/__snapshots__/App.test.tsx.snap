// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app component 1`] = `
<h1>
  Hello world!
</h1>
`;
